specs/altair/light-client/light-client.md

```diff
@@ -0,0 +1,37 @@
+# Altair Light Client -- Light Client
+
+**Notice**: This document is a work-in-progress for researchers and implementers.
+
+## Table of contents
+
+<!-- TOC -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Light client sync process](#light-client-sync-process)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+## Introduction
+
+This document explains how light clients MAY obtain light client data to sync with the network.
+
+## Light client sync process
+
+1. The light client MUST be configured out-of-band with a spec/preset (including fork schedule), with `genesis_state` (including `genesis_time` and `genesis_validators_root`), and with a trusted block root. The trusted block SHOULD be within the weak subjectivity period, and its root SHOULD be from a finalized `Checkpoint`.
+
```

Comment:
I see you use newlines for readability. I'm fine with either way.

/cc @djrtwo in case you have objection :p

Reply:
Not sure I understand correctly, but there is no newline within the paragraph, similar to the rest of the spec. GitHub wraps the lines here (and displays the entire paragraph as line 23).

Reply:
Or, you mean, between the paragraphs?

Reply:
There are multiple conflicting styles within the spec w.r.t. enumerations:

- Newline between items but not within a paragraph, for example in `specs/phase0/weak-subjectivity.md`, or here
- No newline between items, but hard newlines within paragraph and leading spaces on each line, for example in `specs/phase0/validator.md`. The column for the line break appears to be different in each of the files, from a somewhat hard 80 in `specs/bellatrix/p2p-interface.md` through 120-ish in other files.

Furthermore, there are a few cases where the items are all introduced with repeated `1.`, `1.`, `1.` instead of sequential `1.`, `2.`, `3.`..., for example, `sync/optimistic.md` or `tests/README.md`.

I'm fine with either here, there doesn't seem to be strict formatting rules for now. Having absolutely no hard newlines, both within an item as well as between them, makes it more difficult to read while editing, but there are more readers than writers.

Reply:
I meant the newline between the paragraphs `1`, `2`, `3`, ... and that would make it has more margin space between paragraphs. For example:
<img width="200" alt="image" src="https://user-images.githubusercontent.com/9263930/180471378-d244868f-31e2-448d-95f4-22b6dacb4501.png">


> multiple conflicting styles

Right, that happened since we have multiple contributors. Currently, we don't plan to align all specs since it would introduce scary diff lines. 😅